client/layout/masterbar/stats.jsx

```diff
@@ -1,48 +1,23 @@
 /**
  * External dependencies
  */
-import React, { PropTypes } from 'react';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
 import Item from './item';
-import siteStatsStickyTabStore from 'lib/site-stats-sticky-tab/store';
 
 export default React.createClass( {
 	displayName: 'MasterbarStats',
 
 	propTypes: {
-		children: PropTypes.node
-	},
-
-	getInitialState() {
-		return {
-			url: siteStatsStickyTabStore.getUrl()
-		};
-	},
-
-	componentDidMount() {
-		siteStatsStickyTabStore.on( 'change', this.handleStatsStickyTabChange );
-	},
-
-	componentWillUnmount() {
-		siteStatsStickyTabStore.off( 'change', this.handleStatsStickyTabChange );
-	},
-
-	handleStatsStickyTabChange() {
-		var url = siteStatsStickyTabStore.getUrl();
-
-		if ( url !== this.state.url ) {
-			this.setState( {
-				url: url
-			} );
-		}
+		children: React.PropTypes.node,
```

Comment:
I cleaned up references to the `siteStatsStickyTabStore` here since we won't need it going forward. Even if we decided to reinstate the current behavior, we'd do it in a redux-y way :)